proposals/records-wip.md

```diff
@@ -13,7 +13,7 @@ class_declaration
 
 struct_declaration
```

Comment:
This spec is very hard to review because it describes only a fragment of the feature.  Please provide a more complete specification, even if you are implementing only part of it at this time.  That is the only way global issues with the specification can be raised and addressed.